src/huggingface_hub/__init__.py

```diff
@@ -46,7 +46,7 @@
 from typing import TYPE_CHECKING
 
 
-__version__ = "0.33.0.dev0"
+__version__ = "0.33.0"
```

Comment:
no need for this change https://github.com/huggingface/huggingface_hub/pull/3160/commits/50dba392cc7f573b3e62cf9c56b7c1ed48ee2986

```suggestion
__version__ = "0.33.0.dev0"
```

Reply:
ah yes ok so let's not forget about it before merging. Converting back to draft in the meantime